admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,82 @@
+===================
+ownCloud User Types
+===================
+
+ownCloud supports six user types.
+These are:
+
+- `Anonymous`_
+- `Guest`_
+- `Standard User`_
+- `Federated User`_
+- `ownCloud Group Administrator`_
+- `ownCloud Administrator`_
+
+The following information is not an in-depth guide, but more of a high-level overview of each type.
+
+Anonymous
+---------
+
+- Is not a regular user.
+- Has access to specific content made available via public links.
+- Can be password-protected (optional, enforced, policy-enforced).
+- Can have an expiration date (optional, enforced, enforced dependent on password).
+- Has no personal space
+- Has no file ownership (ownership of uploaded/created files is directed to sharer).
+- Has no use of clients.
+- Quota is that of the sharer.
+- Permissions are those granted by the sharer for specific content, e.g., *view-only*, *edit*, and *no file listing*.
+- Can only use apps such file and viewer apps, such as PDF, and Collabora view/edit.
+
+Guest
+-----
+
+- Is a regular user with restricted permissions (identified via e-mail address).
+- Has no personal space 
+- Has no file ownership (ownership of uploaded/created files is directed to sharer).
+- Has access to shared space. The permissions are granted by the sharer.
+- Is not bound to the 'inviting user'.
+- Can log in as long as shares are available.
+- Becomes deactivated when no shares are left ('Shared with guests' filter).
```

Comment:
The "Shared with guests" filter makes it easy for sharers to view and remove _their_ shares with a guest thereby also 'removing' their responsibility regarding this guest. Maybe other users still have shares with that guest. Only when all shares are removed, the guest is disabled and can't log in anymore. Without this information the hint in parentheses is not understandable.

Reply:
Thanks for clarifying that point.